### PR TITLE
latest development version (Lev)

### PR DESCRIPTION
**Pull request by livnev**

### CODE REVIEW

**Comment by livnev** [comment]
This can be merged.

**Review by MrChico — APPROVED** [review]
Seems uncontroversial and fine to me

File: passing.manifest [hunk 1]
```diff
@@ -113,4 +113,7 @@ DaiMove_hope_succ
 DaiMove_nope_succ
 DaiMove_move_succ
 DaiMove_move_fail
-Flapper_dai_succ
\ No newline at end of file
+Flapper_dai_succ
```
Comment (rainbreak) [inline passing.manifest:116]: when `pass`? :crying_cat_face:

Comment (livnev) [reply passing.manifest:116]: That change is included in a klab PR which is not ready yet :crying_cat_face:

**Pull request merged**